Lib/test/test_sysconfig.py

```diff
@@ -296,7 +296,11 @@ def test_user_similar(self):
                 base = base.replace(sys.base_prefix, sys.prefix)
             if HAS_USER_BASE:
                 user_path = get_path(name, 'posix_user')
-                self.assertEqual(user_path, global_path.replace(base, user, 1))
+                expected = global_path.replace(base, user, 1)
+                # bpo-44860: platlib of posix_user doesn't use sys.platlibdir
+                if name == 'platlib' and sys.platlibdir != 'lib':
```

Comment:
What about this? The replacement can always happen.

```suggestion
                if name == 'platlib':
```

Reply:
str.replace() is not reliabe, the full path may contain the ``/lib/`` substring multiple times. Like ``/tmp/lib/runtests/lib/python3.9/``. I prefer to only use it if it's really needed :-p

Or maybe we should use a more reliable replacement, like a regex?